site-scrapers/PediatricAssociatesOfGreaterSalem.js

```diff
@@ -0,0 +1,188 @@
+const https = require("https");
+const moment = require("moment");
+const sites = require("../data/sites.json");
+
+// Set to true to see responses in the console.
+const DEBUG = false;
+
+// Total days to search in the schedule.
+const DAYSTOSEARCH = 16;
```

Comment:
Where did you come up with this number?

Reply:
The requesting website asked for about this many days. 
In testing I went up to 150 days with no days available.
We can basically choose whatever we want here.

Would it be better to parameterize this variable into sites.json?
Or is some other value that you're using (45 e.g. Lowell General) when given the chance?

Reply:
Hm.... I haven't thought about it too too much but I wonder if at least 4 weeks make sense in case people are looking to book a second dose? Thoughts?

Reply:
My inclination is to mimic the original website's request. It would be weird for the scraper to have access to a date that the original website does not.
But it's hard to answer this question well right now as there is not a working example.